### DISCUSSION TITLE
[@types/shippo] Missing method and incorrect constructor

### DISCUSSION THREAD

**Discussion by acommodari** [DefinitelyTyped/DefinitelyTyped#67776] [Issues with a @types package] [answered]
In the `@types/shippo` package it complains that the constructor `new Shippo()` is invalid and must contain at least one argument. This is incorrect as you can have no argument and then use the method `setToken` to add one after the fact.

On the topic of `setToken` even though it exists inside the original js package there are no types for it and thus we get an error when we try to use it.

[a comment by an automated account is not shown]

**Comment by clalexander** [comment]
Those typings are definitely missing/incorrect. Creating a PR to fix

**Comment by clalexander** [comment, marked as answer]
PR [#67797](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67797)

**Reply by acommodari** [reply]
Amazing thank you so much 🙏
